release-notes/docs/3.13.0.md

```diff
@@ -0,0 +1,85 @@
+# 3.13.0 Release Notes
+
+- [Known issues](#known-issues)
+- [Upgrade notes](#upgrade-notes)
+- [Breaking changes](#breaking-changes)
+- [UI/UX changes](#uiux-changes)
+- [Highlights](#highlights)
+- [And more...](#and-more)
+
+## Known issues
+
+None.
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+- [#7235](https://github.com/medic/cht-core/issues/7235): Make it possible to display previous task due date formatting via configuration
```

Comment:
Can we say something other than "previous" here, so it's clear we're providing two good options and not just reverting back. Feel free to edit the issue title and run the script to regenerate this so they're in sync.

Please also make it clear that the default hasn't changed so unless you opt-in nothing will change in the UI.

Also, pop in a screenshot and a link to the configuration documentation so people know what UI change to expect.